src/SFA.DAS.ApplyService.Database/Tables/GatewayAnswer.sql

```diff
@@ -0,0 +1,23 @@
+﻿CREATE TABLE [dbo].[GatewayAnswer](
+	[Id] [uniqueidentifier] NOT NULL,
+	[ApplicationId] [uniqueidentifier] NOT NULL,
```

Comment:
Might be a good idea to index Application Id and Page Id

Reply:
Excellent point.  We also use it coupled tightly with Status to return stuff, so I may create a covered index that uses these two, but stores status too

Reply:
index now added